package.json

```diff
@@ -13,7 +13,7 @@
     "official"
   ],
   "dependencies": {
-    "mongodb-core": "mongodb-js/mongodb-core#6510d7d3d82d84cc0811a853355deaa918b96941"
+    "mongodb-core": "mongodb-js/mongodb-core#NODE-1187"
```

Comment:
We need to make sure not to merge this in

Reply:
Yep! Just wanted to prove it works in CI until we can merge the dependent change in to 3.0.0 over there